src/azul/lambdas.py

```diff
@@ -106,20 +106,55 @@ def _lambda(self):
         return aws.lambda_
 
     def list_lambdas(self) -> list[Lambda]:
+        # Note that this method returns the $LATEST version, which is what
+        # Amazon calls the "unpublished" version.
         return [
             Lambda.from_response(function)
             for response in self._lambda.get_paginator('list_functions').paginate()
             for function in response['Functions']
         ]
 
+    def delete_older_function_versions(self,
+                                       function_name: str,
+                                       keep_version: str) -> None:
```

Comment:
```suggestion
                                       keep_version: int) -> None:
```

Always convert to the canonical representation as soon as possible, at the perimeter of the system/module/program.